src/Admin/AbstractAdmin.php

```diff
@@ -3295,7 +3295,7 @@ protected function configureBatchActions($actions)
      *
      * @deprecated Use configureTabMenu instead
      */
-    protected function configureSideMenu(ItemInterface $menu, $action, ?AdminInterface $childAdmin = null)
```

Comment:
is this change allowed?

Reply:
@jordisala1991 we are using 7.2, so I would say it is ok since inheriting classes can still omit the type declaration.